test/gui/shared/steps/steps.py

```diff
@@ -516,16 +524,16 @@ def step(context, resource, role):
 
 @When('the user logs out of the client-UI')
```

Comment:
just seeing now, the "logout" is of-course also per account, so we should define out of which account (if multiple are configured) the users logsout. Can be a separate PR

Reply:
Yes, you're right about this. I will make a new PR for that.